core/src/main/java/bio/terra/pearl/core/service/survey/SurveyResponseService.java

```diff
@@ -231,7 +238,11 @@ private Answer createAnswer(Answer answer, SurveyResponse response,
         answer.setCreatingParticipantUserId(ppUser.getParticipantUserId());
         answer.setSurveyResponseId(response.getId());
         answer.setSurveyStableId(survey.getStableId());
-        answer.setSurveyVersion(survey.getVersion());
+        if (answer.getSurveyVersion() == 0) {
```

Comment:
I'm not sure if it will be the same logic in the future, which is why I didn't DRY this.  Essentially, how we treat updates and how we treat fresh answers might diverge when it comes to how we assign the version.